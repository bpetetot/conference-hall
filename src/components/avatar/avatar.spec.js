/* eslint-env jest */
import snap from 'tests/snapshot'
import Avatar from './index.js'

const snapshot = props => snap(Avatar)({ ...props })

describe('components/avatar', () => {
  it('should not render when no props', snapshot({}))
  it('should render with initial when no photo', snapshot({ displayName: 'Benjamin' }))
  it('should render with photo', snapshot({ displayName: 'Benjamin', photoURL: 'https://p.url' }))
})
