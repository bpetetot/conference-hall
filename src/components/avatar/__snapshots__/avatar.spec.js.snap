// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/avatar should not render when no props 1`] = `null`;

exports[`components/avatar should render with initial when no photo 1`] = `
<div
  className="cc-avatar-wrapper"
>
  <div
    className="cc-avatar cc-avatar-initials"
    style={
      Object {
        "background": "#42910e",
      }
    }
  >
    <span>
      B
    </span>
  </div>
</div>
`;

exports[`components/avatar should render with photo 1`] = `
<div
  className="cc-avatar-wrapper"
>
  <div
    className="cc-avatar cc-avatar-image"
    style={
      Object {
        "background": "transparent",
      }
    }
  >
    <img
      alt="Benjamin"
      src="https://p.url"
    />
  </div>
</div>
`;
